srcpkgs/pcem/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'pcem'
+pkgname=pcem
+version=17
+revision=1
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
+build_style=gnu-configure
+configure_args="--enable-alsa --enable-release-build --enable-networking --prefix=/usr"
+hostmakedepends="automake pkg-config"
+makedepends="SDL2-devel alsa-lib-devel libopenal-devel wxWidgets-devel"
+short_desc="Emulator for old x86 PCs"
+maintainer="ajshell1 <ajshell1@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://pcem-emulator.co.uk/"
+distfiles="https://pcem-emulator.co.uk/files/PCemV${version}Linux.tar.gz"
+checksum=5b24cb5ce886ed53232385f46594146ba3f7d7eecda90f82892b2dce1cb2f1a4
+
+pre_configure() {
+autoreconf
+}
```

Comment:
Please respect the indentation. And I believe we use the `-fi` flags for autoreconf as well.

Reply:
I've discovered that removing this section entirely fixes the compilation on most ARM platforms, so I've chosen to remove it entirely.